sql/statistics.go

```diff
@@ -55,7 +55,9 @@ type HistogramBucket struct {
 	UpperBound float64 // inclusive
 	Frequency  float64
 }
-
+type StatisticsIndex interface {
+	RowCount() uint64
+}
```

Comment:
This would make more sense right after `StatisticsTable` – please keep them next to each other since they are similar. 

Both the interface itself and the method in it need some godocs to describe them.